paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -84,6 +85,9 @@
 DEFAULT_SLACK_CHANNEL = "#deploy"
 DEFAULT_STUCK_BOUNCE_RUNBOOK = "y/stuckbounce"
 
+TIME_BEFORE_FIRST_DIAGNOSIS = 300
+TIME_BETWEEN_DIAGNOSES = 60
```

Comment:
I am wondering if we should make these configurable either in the system_paasta_config or in the MFD args.  60s seems decent to me, but it's possible it'll increase the load on the api and we'll want to decrease?

Reply:
Yeah, that's probably wise. I should also probably make the polling interval in `check_if_instance_is_done` and the number of threads in the polling threadpool configurable.